src/Payments.sol

```diff
@@ -1175,6 +1191,22 @@ contract Payments is
     ) internal view returns (bool) {
         return block.number > payer.lockupLastSettledAt + rail.lockupPeriod;
     }
+
+    function _zeroOutRail(Rail storage rail) internal {
+        rail.token = address(0);
+        rail.from = address(0); // This now marks the rail as inactive
+        rail.to = address(0);
+        rail.operator = address(0);
+        rail.arbiter = address(0);
+        rail.paymentRate = 0;
+        rail.lockupFixed = 0;
+        rail.lockupPeriod = 0;
+        rail.settledUpTo = 0;
+        rail.terminationEpoch = 0;
```

Comment:
I think you can do this as follows:

```solidity
Rail storage emptyRail;
rail = emptyRail
```

But I'm not 100% sure? You'll probably need to clear the queue at least but... I _think_ the queue should be empty at this point so I'd just assert that instead.

Reply:
https://github.com/FilOzone/payments/issues/36